lib/conjur/debify.rb

```diff
@@ -120,30 +120,30 @@ def login_to_registry(appliance_image_id)
 command "clean" do |c|
   c.desc "Set the current working directory"
   c.flag [ :d, "dir" ]
-    
+
```

Comment:
It's not so great to have a bunch of extra whitespace in the file, but mixing these changes in with more substantive changes makes it harder to review.

Reply:
good point, my bad, vscode did this automatically on save i guess and i didn't notice. will definitely keep an eye out for this next time. i'll turn the trim whitespace setting off